src/data/showcases.js

```diff
@@ -1399,7 +1399,16 @@ export const Showcases = [
     website: "https://arpradio.media",
     source: null,
     tags: ["minting", "ecosystem", "metadata" ]
-  }
+  },
+  {
+    title: 'Spidex',
+    description:
+      'The AI layer unifying Cardano DeFi. Spidex provides DEX aggregation, collective AI agents, and conversational interface for seamless cross-platform trading with optimized fees and aggregated liquidity.',
+    preview: require('./showcase/spidex.png'),
+    website: 'https://spidex.ag/',
+    source: null,
+    tags: ['analytics', 'dex'],
```

Comment:
```suggestion
    tags: [dex'],
```
This tag is for blockchain analytics: not analytics in general.  If you believe you're providing general analytics then would you please explain here?

Reply:
Ah got it. We're specifically providing blockchain analytics, not general analytics: 
- Token price analysis and trends (via Taptools, Dexhunter, Minswap)
- On-chain wallet portfolio tracking
- Transaction history analysis
- Token holder distribution and top traders
- DEX liquidity and trading volume metrics
- Smart money flow tracking

Reply:
@hiiamtrong it's my understanding that these analytical features are _not_ the general blockchain analytics offered by portfolio items with the `analytics` tag.  Maybe @0xBora could offer a second opinion if appropriate since he's just done a major overhaul of the categorisation.